cmd/up/exec.go

```diff
@@ -214,6 +214,10 @@ func (eg *envsGetter) getEnvs(ctx context.Context) ([]string, error) {
 
 	envs = append(envs, eg.getDefaultLocalEnvs()...)
 
+	for _, env := range eg.dev.Environment {
+		envs = append(envs, fmt.Sprintf("%s=%s", env.Name, env.Value))
```

Comment:
Did we check what happens if the same variable is defined in multiple sections? I mean if I have one environment set in the pod but when I run `okteto up` with a manifest that is overwriting it, is it working as expected?
Could we add a test for that scenario?

Reply:
then variable that is set using `environment` section is the one used. Same approach described in docs https://www.okteto.com/docs/reference/manifest/#environment-string-optional:~:text=Environment%20variables%20declared%20in%20the%20environment%20section%20override%20these%20values.

Reply:
I will add a test covering it

Reply:
b2da3c8657915cfb17cd8bba0b05b369fa0a2928 @jLopezbarb test to check overwrites